CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.8.0] - 2016-09-26
```

Comment:
Oh, you also need to remember to add the line to the bottom of the file to make that `[…]` a valid target. (Sorry for not spotting that first time around).